pumpkin-world/assets/no_bench_no_blend_0_0.java

```diff
@@ -0,0 +1,158 @@
+package com.example.mixin;
```

Comment:
What the hell is this class and why is this in pumpkin-world ?

Reply:
Its code to generate the no_bench_no_blend_0_0.chunk file

Reply:
Why you don't put it into the extractor/ ?

Reply:
I didn't think about that, I'll do that